examples/run_query_async.py

```diff
@@ -21,7 +20,7 @@ def run(database: str, engine: str, command: str, readonly: bool, profile: str):
     cfg = config.read(profile=profile)
     ctx = api.Context(**cfg)
     rsp = api.query_async(ctx, database, engine, command, readonly=readonly)
-    show.results(rsp, "multipart")
+    show.results(rsp, "multipart" if isinstance(rsp, list) else "wire")
```

Comment:
So if it's not in multipart, we conclude that it's in json format? Does this mean that if we set the sync_mode to be true, the result is in JSON and if not, it is in multipart, no?